README.md

```diff
@@ -3,19 +3,33 @@
 ## Installation with pip
```

Comment:
PyPI does not show top-level README (https://test.pypi.org/project/kentik-api/). So it does not show installation instruction. Should't we push whole repository to PYPI?

Reply:
Yes, we should share the test and (eventually) prod versions of the top-level.  Thank you @Fenthick

Reply:
let's discuss it with Maciej and do in separate PR